ikarus/controlRoutines/pathFollowingTechnique.hh

```diff
@@ -24,11 +24,13 @@ namespace Ikarus {
   requires Concepts::PathFollowingStrategy<PathFollowingType, typename NonLinearSolver::NonLinearOperator>
   class PathFollowing : public IObservable<ControlMessages> {
   public:
-    PathFollowing(const std::shared_ptr<NonLinearSolver>& p_nonLinearSolver, int loadSteps, double stepSize,
-                  PathFollowingType p_pathFollowingType = Ikarus::StandardArcLength{})
+    PathFollowing(const std::shared_ptr<NonLinearSolver>& p_nonLinearSolver, int loadSteps, double stepSize, bool p_isAdaptiveStep,
```

Comment:
I would suggest that we provide here a function that takes care about the correct automatic stepsizing

Reply:
Done